### DISCUSSION TITLE
backup: clean  AWS(S3) not clean any files

### DISCUSSION THREAD

**Discussion by elmarzouguidev** [spatie/laravel-backup#1725] [Q&A]
backup: clean to AWS(S3) not clean any files

Starting cleanup...
Cleaning backups of appName on disk s3...
Used storage after cleanup:  X MB.
Cleanup completed!

but all files still in S3
*******************************************************
"laravel/framework": "^10.0",
 "league/flysystem-aws-s3-v3": "^3.0"
 "spatie/laravel-backup": "^8.1"

**Comment by elmarzouguidev** [comment]
and also in local disk

**Comment by xHeaven** [comment]
Do you have your configuration set up properly as per your needs? Refer to the documentation: https://spatie.be/docs/laravel-backup/v8/cleaning-up-old-backups/overview

**Comment by elmarzouguidev** [comment]
@xHeaven yes i have the right config
